src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWPApplication.cs

```diff
@@ -679,8 +681,8 @@ private BitmapImage ImageFromPath(string path)
             {
                 MemoryStream memoryStream = new MemoryStream();
 
-                byte[] fileBytes = File.ReadAllBytes(path);
-                memoryStream.Write(fileBytes, 0, fileBytes.Length);
+                var fileStream = File.OpenRead(path);
+                fileStream.CopyTo(memoryStream);
```

Comment:
dispose filestream, or add into using

Reply:
Good catch!
We should add a Analyzer to error on the build if we (on accident) forget it.